src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/block/pool-detail/pool-detail.component.ts

```diff
@@ -13,18 +13,21 @@ import { PoolService } from '../../../shared/services/pool.service';
   styleUrls: ['./pool-detail.component.scss']
 })
 export class PoolDetailComponent implements OnInit, OnDestroy {
-
   name: string;
   images: any;
   columns: any;
```

Comment:
Use "CdTableColumn[]" as type instead of any.

Reply:
done